data/discovery/field/clinical-commissioning-group.yaml

```diff
@@ -0,0 +1,6 @@
+cardinality: '1'
+datatype: string
+field: clinical-commissioning-group
+phase: discovery
+register: clinical-commissioning-group
+text: The NHS code assigned to the CCG by the Organisation Data Service (part of NHS Digital).
```

Comment:
I suggest @psd review the clinical-commissioning-group description. He could confirm whether we should state the name of the organisation currently assigning a code.

I suggest @J-Lambo the text. She could confirm whether we should expand acronyms and remove parentheses.